### ISSUE TITLE
[HOLD for payment 2024-02-07] [$500] [ECARD TRANSACTIONS] report sub-total says Out of pocket, instead of Out-of-pocket spend (missing hyphen, and "spend")

### ISSUE THREAD

**Issue by kevinksullivan** [Expensify/App#34628]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue reported by:** @kevinksullivan 
**Slack conversation:** https://expensify.slack.com/archives/C05DWUDHVK7/p1705442308576449

## Action Performed:
1. Create a free workspace and set it as your default
2. Incur a charge on an Expensify Card
3. Add a manual request
4. Tap into the report containing both expenses

## Expected Result:
The sub-total should show `Out-of-pocket spend`, with the reimbursable total.

## Actual Result:
The subtotal shows `Out of pocket`, and is missing the hyphens, and "spend".

![image](https://github.com/Expensify/App/assets/22508290/2da2d77b-b0c4-4ca3-bdca-5107f979b8b6)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014c5b9f85422667c6</li>
        <li>Upwork Job ID: 1748142102095720448</li>
        <li>Last Price Increase: 2024-01-19</li>
<li>Automatic offers: </li>
<ul>
<li>Ollyws | Contributor | 28109141</li>
</ul></li>
    </ul>
</details>

**Comment by grgia** [comment]
@Ollyws I went ahead and assigned this one out since it's simple

**Comment by grgia** [comment]
I don't think we need C+ review for this though since it's a copy change

**Comment by kevinksullivan** [comment]
Waiting on @grgia 's review

**Comment by dylanexpensify** [comment]
Where's the new PR mentioned in closed PR @AnshuAgarwal24

**Comment by dylanexpensify** [comment]
https://github.com/Expensify/App/pull/34790#issuecomment-1910190115

**Comment by pecanoro** [comment]
@dylanexpensify Right above your comment 😄 

<img width="954" alt="Monosnap  $500   ECARD TRANSACTIONS  report sub-total says Out of pocket, instead of Out-of-pocket spend (missing hyphen, and  spend ) ·… 2024-01-26 14-05-41" src="https://github.com/Expensify/App/assets/6474442/82314308-c649-469b-80d1-1240775c9a20">

**Comment by pecanoro** [comment]
@AnshuAgarwal24 7 days after hitting production.
